README.md

```diff
@@ -371,6 +376,87 @@ Failed: text 'foo' not present on element with locator 'By(css selector, h1)'.
+
+#### How I know that my code is ok?
+
+You will know that you have solved all your problems when don't see any warning on console like this:
+
+```
+Protractor-helper warning: Function 'getBodyElementFromCurrentBrowserOrBrowserInstance' will be deprecated in version 4.0.0! [Read more on www.npmjs.com/package/protractor-helper#preparation-to-next-major-version]
+```
+
+### Functions that will be replaced
+
+The 8 below will be replaced to reduce complexity, with the new functions with less arguments and smaller name.
+
+| Old                                                                                     | New                                                               |
+| --------------------------------------------------------------------------------------- | ----------------------------------------------------------------- |
+| [clickWhenClickable](#clickWhenClickable)                                               | [click](#click)                                                   |
+| [fillFieldWithTextWhenVisible](#fillFieldWithTextWhenVisible)                           | [fillFieldWithText](#fillFieldWithText)                           |
+| [fillInputFieldWithFileWhenPresent](#fillInputFieldWithFileWhenPresent)                 | [uploadFileIntoInputField](#uploadFileIntoInputField)             |
+| [clearFieldWhenVisible](#clearFieldWhenVisible)                                         | [clear](#clear)                                                   |
+| [clearFieldWhenVisibleAndFillItWithText](#clearFieldWhenVisibleAndFillItWithText)       | [clearFieldAndFillItWithText](#clearFieldAndFillItWithText)       |
+| [tapWhenTappable](#tapWhenTappable)                                                     | [tap](#tap)                                                       |
+| [fillFieldWithTextWhenVisibleAndPressEnter](#fillFieldWithTextWhenVisibleAndPressEnter) | [fillFieldWithTextAndPressEnter](#fillFieldWithTextAndPressEnter) |
+| [scrollToElementWhenVisible](#scrollToElementWhenVisible)                               | [scrollToElement](#scrollToElement)                               |
+
+#### What to do?
+
+- Only change the function on **Old** column to the matching function of **New** column, removing the `errorMessage` argument.
+  - All new methods, unlike the old ones, don't use the `errorMessage` argument.
+
+#### How I know that my code is ok?
+
+You will know that you have solved all your problems when don't see any warning on console like this:
+
+```
+Protractor-helper warning: Function 'fillFieldWithTextWhenVisible' will be deprecated in version 4.0.0! Please use the new 'fillFieldWithText' function instead! [Read more on www.npmjs.com/package/protractor-helper#preparation-to-next-major-version]
+```
+
+### Argument `errorMessage` will be removed
+
+We will remove all `errorMessage` argument from the library because we trust on our messages. They are clear and tell you exactly why the test has failed.
```

Comment:
What about: `We will remove all the `errorMessage` arguments in favor of the messages provided by the library, which are clear and tell exactly why the test has failed.` ?